### ISSUE TITLE
Choose a host

### ISSUE THREAD

**Issue by reefdog** [PhilanthropyDataCommons/data-viewer#8]
We'll need to host this somewhere!

Currently and for the foreseeable future, the site needs no back end. It can be built for fast and inexpensive static delivery via a CDN. We're agnostic, but think [Netlify](https://www.netlify.com/) is a good candidate for its reliability, price, and excellent GitHub integration (such as easy CD).

We'll have a discussion in the team meeting today to find out any outside requirements (stakeholder/project commitments, existing contracts/platforms, etc.).

**Comment by jasonaowen** [comment]
During today's meeting, we decided to move forward with Netlify, with the understanding that if we encountered a blocking problem it would be easy to switch to something else; this is slightly stronger than a proof-of-concept in that we intend to keep using it if it works. (Which I guess is how proofs-of-concept _tend_ to go, but nominally they're supposed to be thrown away!)

We'll use `pilot.philanthropydatacommons.org` for hosting the static assets from this front-end repo, and we'll move the back-end to `api.philanthropydatacommons.org`, which is where we want it to live long-term.

I will set up a Netlify account, start setting up the subdomain, and coordinate with Jonathan to make DNS changes.
